pkg/validation/policy/validate.go

```diff
@@ -648,7 +648,7 @@ func isLabelAndAnnotationsString(rule kyvernov1.Rule) bool {
 				if ok {
 					// range over labels
 					for _, val := range labelKey {
-						if reflect.TypeOf(val).String() != "string" {
+						if val == nil || reflect.TypeOf(val).String() != "string" {
```

Comment:
Is it possible to have a test case for this?

Reply:
I was thinking the same ! Thanks for asking =)

Reply:
Done